GiniVision/Classes/GiniVision.swift

```diff
@@ -87,12 +87,20 @@ import UIKit
      - note: Screen API only.
      
      - parameter delegate: An instance conforming to the `GiniVisionDelegate` protocol.
+     - parameter importedFile:  A document which comes from other source different than CameraViewController (optional)
      
      - returns: A presentable navigation view controller.
      */
-    public class func viewController(withDelegate delegate: GiniVisionDelegate) -> UIViewController {
-        let cameraContainerViewController = CameraContainerViewController()
-        let navigationController = GiniNavigationViewController(rootViewController: cameraContainerViewController)
+    public class func viewController(withDelegate delegate: GiniVisionDelegate, importedFile:Data? = nil) -> UIViewController {
+        let viewController:UIViewController = {
```

Comment:
I like this approach as a programmer. As a maintainer or contributor I would avoid it. The parantheses are easy to miss and you have to go back and check again after you see them and understand that this is a block call for conditional instantiation. Also imho a block is a bit of an overkill for this case. I don't like being a party 💩-er, but in this case I would vote for a simple approach:
```
let viewController:UIViewController;
if let importedFile = importedFile {
    viewController = ReviewContainerViewController(imageData: importedFile)
} else {
    viewController = CameraContainerViewController()
}
```

Reply:
There is no discussion. I don't know way I chose the difficult approach, it is much simpler what you suggest.
No `;` needed btw ☕ 😉